src/Components/GraduatesList/GraduatesList.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import { Checkbox, Input, Table } from "semantic-ui-react";
+import FilterButton from "./FilterButton";
+import Graduated from "./Graduated";
+import axios from "axios";
+
+
+class GraduatesList extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			graduates: [],
+			filterBy: 'All',
+			filterCriteria:'All'
+		};
+	}
+
+	componentDidMount() {
+		const API_URL = process.env.REACT_APP_API_URL;
+	  axios
+			.get(`${API_URL}/empresas-egresades`)
+			.then(response => {
+				const graduates = response.data;
+				this.setState({ graduates });
+			})
+			.catch(function(error) {
+				console.log(error);
+		});
+	}
+
+	listGraduates=() => {
+		if (this.state.filterCriteria === 'All')
+			return this.MapGraduates(this.state.graduates);
+		else
+			return this.MapGraduates(this.factoryFilter())
+	}
+	
+	MapGraduates(listEgresades){
+		return(
+			listEgresades.map((currentList, i) => {
+				return <Graduated item={currentList} key={i} />
+		}));
+	}
+
+	factoryFilter(){
```

Comment:
We are gonna implement a switch case for avoid duplicity
```javascript
 factoryFilter(){
  switch (this.state.filterBy) {
    case 'ModuleCompleted':
      return this.filterByModuleCompleted(this.state.filterCriteria);		
    case 'EnglishLevel':
      return this.filterByEnglishLevel(this.state.filterCriteria);				
    case 'Node':
      return this.filterByNode(this.state.filterCriteria);		
    default:
      return this.state.graduates;
    }
 }
````
just a suggestion 😄